concourse/pipelines/templates/resources/release-tarball-artifact-tpl.yml

```
@@ -0,0 +1,20 @@
+{# Template for a PXF build artifact resource definition.
+ #
+ # Expects a dictionary with the name 'x' to be set as defined in macros.j2 file
+ #}
+
+{# include release tarball artifacts only if the test os is centos to avoid duplicates #}
+{% if x.test_os == 'centos' %}
+{# release tarball artifacts are only needed for gp5 and gp6, we produce only rpms for gp7 #}
+{% if x.gp_ver != '7' %}
```

Any reason to not combine the separate `if` conditions into a single one?
```suggestion
{# release tarball artifacts are only needed for gp5 and gp6, we produce only rpms for gp7 #}
{% if x.test_os == 'centos' and x.gp_ver != '7' %}
```